app/migrations/20240809141227-create-datasourceI18n.cjs

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+const sql_up = `create table if not exists public."DataSourceI18n"
+(
+    datasource_id uuid not null constraint "DataSourceI18n_pkey" primary key,
+    datasource_name            varchar(255),
+    "URL"                      varchar(255),
+    dataset_description        jsonb,
+    access_type                varchar(255),
```

Comment:
Let's use `text` over `varchar(255)`. Varchar is annoying to deal with once the values start going over this limit and it's not really that much more performant in modern Postgres.